### PR TITLE
Replaced Force Directed Graph Challenge

### LINKED ISSUE #8722: Show Relationships with a Force Directed Graph broken after deprecated Camper news API.
Challenge [Show Relationships with a Force Directed Graph](https://www.freecodecamp.com/challenges/show-relationships-with-a-force-directed-graph) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

### PR DESCRIPTION
**Pull request by DealPete**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [X] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #8722
#### Description

The challenge "Show Relationships with a Force Directed Graph" asks campers to build a force directed graph based off relationships derived from the Camper News Feed. The Camper News Feed was recently removed, breaking existing solutions to the challenge.
This pull request replaces the challenge with a new one based on a dataset hosted on github.
The new challenge, "Show National Contiguity with a Force Directed Graph", uses countries as nodes, and these nodes are linked if the countries share a national border. The flag icons are loaded from a spritesheet generated at https://www.flag-sprites.com/. The dataset, hosted at https://github.com/DealPete/forceDirected, was adapted from a dataset downloaded from the Correlates of War Project (http://www.correlatesofwar.org/).
No libraries other than d3 were used.

### CODE REVIEW

File: seed/challenges/02-data-visualization-certification/data-visualization-projects.json [hunk 1]
```diff
@@ -119,18 +119,16 @@
     },
     {
       "id": "bd7198d8c242eddfaeb5bd13",
-      "title": "Show Relationships with a Force Directed Graph",
+      "title": "Show National Contiguity with a Force Directed Graph",
       "description": [
-        "<strong>Objective:</strong> Build a <a href='https://codepen.io' target='_blank'>CodePen.io</a> app that is functionally similar to this: <a href='https://codepen.io/FreeCodeCamp/full/KVNNXY' target='_blank'>https://codepen.io/FreeCodeCamp/full/KVNNXY</a>.",
+        "<strong>Objective:</strong> Build a <a href='https://codepen.io' target='_blank'>CodePen.io</a> app that is functionally similar to this: <a href='https://codepen.io/FreeCodeCamp/pen/xVopBo' target='_blank'>https://codepen.iohttps://codepen.io/FreeCodeCamp/pen/xVopBo</a>.",
```
Comment (theflametrooper) [inline seed/challenges/02-data-visualization-certification/data-visualization-projects.json]: ~~https://codepen.iohttps://codepen.io/FreeCodeCamp/pen/xVopBo~~ https://codepen.io/FreeCodeCamp/pen/xVopBo

**Comment by theflametrooper** [comment]
cc/ @FreeCodeCamp/issue-moderators @QuincyLarson

File: seed/challenges/02-data-visualization-certification/data-visualization-projects.json (hunk 1 above)
Comment (raisedadead) [inline seed/challenges/02-data-visualization-certification/data-visualization-projects.json]: @theflametrooper @DealPete 
It should actually be : `https://codepen.io/FreeCodeCamp/`**full**`/xVopBo`

**Comment by raisedadead** [comment]
@DealPete Thanks for the PR, please make the changes, amend the commit and we are good to go.

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @DealPete ! Lots of hard work has gone into this.

**Pull request merged**